docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<p>Originally, the helper functions implementation was hidden in <code>finance_notebook_helper</code>. Is the idea now to show some of the implementation to run the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>correct - we chose to pull them out to show users exactly what was going on for them to fully replicate them if needed</p>